addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -47,15 +46,20 @@ const StorePatch = {
             model: "mail.box",
             name: _t("History"),
         };
-        try {
-            // useful for synchronizing activity data between multiple tabs
-            this.activityBroadcastChannel = new browser.BroadcastChannel("mail.activity.channel");
-            this.activityBroadcastChannel.onmessage =
-                this._onActivityBroadcastChannelMessage.bind(this);
-        } catch {
-            // BroadcastChannel API is not supported (e.g. Safari < 15.4), so disabling it.
-            this.activityBroadcastChannel = null;
-        }
+        this.env.services["multi_tab"].subscribe("mail.activity/insert", ({ detail }) => {
```

Comment:
Since we're not providing a way to unsubscribe and even if we did, it would be good to directly pass the parameter to the callback instead of destructuring it every time.

Reply:
It seems like we often use this approach in `mail`, what are the benefits of not destructuring ? I personnaly think it looks nicer like this but may be missing something.

Reply:
Destructing is not the issue. I was suggesting to do it in the `subscribe` method to avoid destructing the event payload in each call site.